docs/setup/pip.md

```diff
@@ -4,35 +4,48 @@ Opting for `pip` is ideal for those with a comprehensive grasp of Python's ecosy
 
 The Khiops binaries must be installed as a prerequisite. This also ensures the installation of the appropriate version of `MPICH` library.
 
-We support :simple-python: **Python from 3.8 to 3.11**. 
+We support :simple-python: **Python from 3.8 to 3.12**. 
 
-=== "Ubuntu/Debian"
+=== "Ubuntu 20-22 / Debian 10"
```

Comment:
Ubuntu 20, 22 / Debian 10
(replace "-" with ", "), as Ubuntu 21 is not an LTS release and is not tested.

Reply:
Done.